syncd/SaiSwitch.cpp

```diff
@@ -62,10 +62,7 @@ SaiSwitch::SaiSwitch(
 
     helperInternalOids();
 
-    if (getSwitchType() == SAI_SWITCH_TYPE_NPU)
-    {
-        helperCheckLaneMap();
-    }
+    helperCheckLaneMap();
```

Comment:
why did the original check of SAI_SWITCH_TYPE_NP should be removed, how it prevented lane mapping for multi-asic?

Reply:
to support port breakout, I think both NPU system and VOQ system need to create the lane mapping.

Reply:
I forget to mention that the modular chassis uses SAI_SWITCH_TYPE_VOQ, not NPU.